src/components/ui/MonacoEditor.svelte

```diff
@@ -146,8 +143,12 @@
 
           // Errors in the dispatch won't trigger the retry and will just fail.
           model = editor?.getModel(); // Set here so parents can bind to the model easily.
-          if (model != null && tsWorker !== null) {
-            dispatch('fullyLoaded', { model, worker: tsWorker });
+          if (editor != null && model != null && tsWorker !== null) {
+            dispatch('fullyLoaded', { editor, model, worker: tsWorker });
+            // Manually trigger a change in the editor to evaluate the source on load
+            // Will highlight errors on load without the user having to input something in the editor
+            editor.trigger('', 'type', { text: ' ' });
+            editor.trigger('', 'deleteLeft', {});
```

Comment:
One odd side effect here is that in read-only mode the editor shows this tooltip
<img width="394" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/9328ffd3-dfca-41a4-a5bf-240f27f6a65e">
which can be seen on page load for scheduling. Any way to do this highlighting only if in edit mode?

Reply:
nice catch! I updated it to only do it when in edit mode.